### PR TITLE
Induction dynamic forms

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes include

1. Induction form - Logic to add a group of custom fields based on activity status - the status should be "Completed".

### CODE REVIEW

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php [hunk 1]
```diff
@@ -125,6 +125,32 @@ function goonjcustom_civicrm_buildForm( $formName, $form ) {
...
+                            if (selectedText === 'Completed') {
+                                customGroup.show();
+								// TODO - need to add logic to handle required fields
+                            } else {
+                                customGroup.hide();
+								// TODO - need to add logic to handle required fields
+                            }
+                        }
+                        $('#status_id').change(function() {
+						var selectedValue = $(this).val();
```
Comment (pokhiii) [inline wp-content/civi-extensions/goonjcustom/goonjcustom.php:145]: Indentation

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php [hunk 2]
```diff
@@ -125,6 +125,32 @@ function goonjcustom_civicrm_buildForm( $formName, $form ) {
...
+                            updateCustomGroupVisibility();
+                        });
+
+                        // Initial check to show/hide the custom group and update required constraints
+                        setTimeout(updateCustomGroupVisibility, 500);
```
Comment (pokhiii) [inline wp-content/civi-extensions/goonjcustom/goonjcustom.php:150]: Is this a hack or something?

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php (hunk 1 above)
Comment (pokhiii) [inline wp-content/civi-extensions/goonjcustom/goonjcustom.php:145]: What is the use of this variable?

File: wp-content/themes/goonj-crm/assets/styles/main.scss [hunk 3]
```diff
@@ -244,4 +244,5 @@ form input {
     .af-field-type-text {
         width: 100% !important;
     }
+
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/assets/styles/main.scss:247]: Not needed

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php (hunk 2 above)
Comment (Satendra-SR) [reply wp-content/civi-extensions/goonjcustom/goonjcustom.php:150]: Yes, it takes some time for form to load, this logic need some delay before it get executed with the available elements.

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php (hunk 1 above)
Comment (Satendra-SR) [reply wp-content/civi-extensions/goonjcustom/goonjcustom.php]: fixed

Comment (Satendra-SR) [reply wp-content/civi-extensions/goonjcustom/goonjcustom.php]: Not needed for now, removed

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php (hunk 2 above)
Comment (pokhiii) [reply wp-content/civi-extensions/goonjcustom/goonjcustom.php:149]: Can you add this as a comment?

**Review by pokhiii — APPROVED** [review]

File: wp-content/civi-extensions/goonjcustom/goonjcustom.php (hunk 2 above)
Comment (Satendra-SR) [reply wp-content/civi-extensions/goonjcustom/goonjcustom.php:149]: Done

**Pull request merged**